### ISSUE TITLE
      throw PlatformException(code: errorCode, message: errorMessage, details: errorDetails);

### ISSUE THREAD

**Issue by Rajdeep-4321** [flutter/flutter#47449]
I am getting this error while trying to delete the auth user...
but I am getting this error

```dart
import 'package:flutter/material.dart';
import 'dart:async';
import 'package:firebase_auth/firebase_auth.dart';

class RemoveUser extends StatefulWidget {
  @override
  State<StatefulWidget> createState() => new _HomePageState();
}

class _HomePageState extends State<RemoveUser> {
  Widget _showButtonList() {
    return new Container(
      padding: EdgeInsets.all(26.0),
      child: new ListView(
        children: <Widget>[

          _removeUserContainer(),
        ],
      ),
    );
  }

  
  _removeUserContainer() {
    return new MaterialButton(
      shape:
      RoundedRectangleBorder(borderRadius: new BorderRadius.circular(30.0)),
      onPressed: () async {
          final FirebaseUser user = await FirebaseAuth.instance.currentUser();
          user.delete().then((_) {
            print("Succesfull user deleted");
          }).catchError((error) {
            print("user can't be delete" + error.toString());
          });
          return null;


      },
      minWidth: MediaQuery
          .of(context)
          .size
          .width,
      padding: EdgeInsets.fromLTRB(20.0, 15.0, 20.0, 15.0),
      color: Colors.red,
      textColor: Colors.white,
      child: Text(
        "Thank You",
        textAlign: TextAlign.center,
      ),
    );
  }
  @override
  Widget build(BuildContext context) {
    return new Scaffold(
      appBar: new AppBar(
        title: new Text('Flutter login demo'),

      ),
      body: _showButtonList(),


    );
  }
}
```

already added fingerprint

**Comment by janmoppel** [comment]
Hello @Rajdeep-4321 !

Thank you for posting an issue.
Can you please add your `flutter doctor -v` command output and the full error log?

**Comment by Rajdeep-4321** [comment]
[√] Android Studio (version 3.5)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 42.0.1
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b03)

[√] Connected device (1 available)
    • Redmi Note 5 • 29e14d1c9805 • android-arm64 • Android 8.1.0 (API 27)

• No issues found!

C:\Users\REGRA\Downloads\flutterAuthentication-master\frost>

**Comment by Rajdeep-4321** [comment]
how to view the full error log?

**Comment by janmoppel** [comment]
Did you add the whole output of the `flutter doctor -v` command? I cannot see the Flutter plugin information.

For the full error log, you can build the application with tool, using this command: `flutter build --verbose`

**Comment by Rajdeep-4321** [comment]
command not working ```flutter build --varbose```

**Comment by Rajdeep-4321** [comment]
yes I have added whole output

**Comment by iapicca** [comment]
> command not working `flutter build --varbose`

please double-check the syntax  `flutter run --verbose`
thank you

**Comment by Rajdeep-4321** [comment]
C:\Users\REGRA\Downloads\flutterAuthentication-master\frost>flutter build --verbose
[  +37 ms] executing: [D:\DROID\flutter\] git -c log.showSignature=false log -n 1 --pretty=format:%H
[  +99 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +1 ms] 27321ebbad34b0a3fafe99fac037102196d655ff
[   +1 ms] executing: [D:\DROID\flutter\] git describe --match v*.*.* --first-parent --long --tags
[  +77 ms] Exit code 0 from: git describe --match v*.*.* --first-parent --long --tags
[        ] v1.12.13+hotfix.5-0-g27321ebba
[  +10 ms] executing: [D:\DROID\flutter\] git rev-parse --abbrev-ref --symbolic @{u}
[  +97 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/stable
[        ] executing: [D:\DROID\flutter\] git ls-remote --get-url origin
[  +68 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
Missing subcommand for "flutter build".


Run 'flutter -h' (or 'flutter <command> -h') for available flutter commands and options.

**Comment by Rajdeep-4321** [comment]
tell me is this code right? then why this is giving error

**Comment by iapicca** [comment]
Hi @Rajdeep-4321 
the command `flutter build` has to be completed
https://flutter.dev/docs/deployment/android#building-the-app-for-release
and in our case adding `--verbose` at the end,
something like `flutter build apk --verbose` or `flutter build appbundle --verbose`

Regardless I'm not sure if it's necessary to actually build anything to investigate the issue,
also running simple `flutter run --verbose ` should give an idea of the error;
I also noticed that you provided a partial `flutter doctor -v`
https://github.com/flutter/flutter/issues/47449#issuecomment-567499610
can you please provide the complete output

thank you

**Comment by VladyslavBondarenko** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. We are therefore reluctantly going to close this bug for now. Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.
Could everyone who still has this problem please file a new issue with the exact descriptions what happens, logs and the output of 'flutter doctor -v'  please.
All system setups can be slightly different so its always better to open new issues and reference related issues.

**Comment by harrykrish-web** [comment]
Hi, i am facing same issue
![screen](https://user-images.githubusercontent.com/60507974/89683880-f750d880-d916-11ea-898d-c5ce113b7583.jpg)

Flutter Doctor Output:
[✓] Flutter (Channel stable, v1.17.5, on Linux, locale en_GB.UTF-8)
    • Flutter version 1.17.5 at /home/harik/snap/flutter/common/flutter
    • Framework revision 8af6b2f038 (5 weeks ago), 2020-06-30 12:53:55 -0700
    • Engine revision ee76268252
    • Dart version 2.8.4

 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
    • Android SDK at /home/harik/Android/Sdk
    • Platform android-30, build-tools 30.0.1
    • Java binary at: /home/harik/installations/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Android Studio (version 4.0)
    • Android Studio at /home/harik/installations/android-studio
    • Flutter plugin version 47.1.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.47.3)
    • VS Code at /usr/share/code
    • Flutter extension version 3.13.2

[✓] Connected device (1 available)
    • SM G960F • 2311939b1b057ece • android-arm64 • Android 10 (API 29)

• No issues found!

**Comment by harrykrish-web** [comment]
> Hi, i am facing same issue
> ![screen](https://user-images.githubusercontent.com/60507974/89683880-f750d880-d916-11ea-898d-c5ce113b7583.jpg)
> 
> Flutter Doctor Output:
> [✓] Flutter (Channel stable, v1.17.5, on Linux, locale en_GB.UTF-8)
> • Flutter version 1.17.5 at /home/harik/snap/flutter/common/flutter
> • Framework revision [8af6b2f](https://github.com/flutter/flutter/commit/8af6b2f038c1172e61d418869363a28dffec3cb4) (5 weeks ago), 2020-06-30 12:53:55 -0700
> • Engine revision ee76268252
> • Dart version 2.8.4
> 
> [✓] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
> • Android SDK at /home/harik/Android/Sdk
> • Platform android-30, build-tools 30.0.1
> • Java binary at: /home/harik/installations/android-studio/jre/bin/java
> • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
> • All Android licenses accepted.
> 
> [✓] Android Studio (version 4.0)
> • Android Studio at /home/harik/installations/android-studio
> • Flutter plugin version 47.1.2
> • Dart plugin version 193.7361
> • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
> 
> [✓] VS Code (version 1.47.3)
> • VS Code at /usr/share/code
> • Flutter extension version 3.13.2
> 
> [✓] Connected device (1 available)
> • SM G960F • 2311939b1b057ece • android-arm64 • Android 10 (API 29)
> 
> • No issues found!

My Code is:

`final GoogleSignIn googleSignIn = GoogleSignIn();

class Home extends StatefulWidget {
  @override
  _HomeState createState() => _HomeState();
}

class _HomeState extends State<Home> {
  bool isAuth = false;

  @override
  void initState() {
    super.initState();
    googleSignIn.onCurrentUserChanged.listen((account) {
      handleSignIn(account);
    }, onError: (err) {
      print('Error SigningIN: $err');
    });

    googleSignIn.signInSilently(suppressErrors: false).then((account) {
      handleSignIn(account);
    }).catchError((err) {
      print('Error SigningIN: $err');
    });
  }

  handleSignIn(GoogleSignInAccount account) {
    if (account != null) {
      print('User signed in: $account');
      setState(() {
        isAuth = true;
      });
    } else {
      setState(() {
        isAuth = false;
      });
    }
  }

  login() {
    googleSignIn.signIn();
  }

  logout() {
    googleSignIn.signOut();
  }

  Widget buildAuthScreen() {
    return RaisedButton(child: Text('logout'), onPressed: logout);
  }

  Scaffold buildUnAuthScreen() {
    return Scaffold(
      body: Container(
        decoration: BoxDecoration(
          gradient: LinearGradient(
            begin: Alignment.topRight,
            end: Alignment.bottomLeft,
            colors: [Colors.teal, Colors.purple],
          ),
        ),
        alignment: Alignment.center,
        child: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          crossAxisAlignment: CrossAxisAlignment.center,
          children: <Widget>[
            Text(
              'Vizag Fish',
              style: TextStyle(
                // fontFamily: "Signatra",
                fontSize: 90.0,
                color: Colors.white,
              ),
            ),
            GestureDetector(
              onTap: login,
              child: Container(
                width: 260.0,
                height: 60.0,
                decoration: BoxDecoration(
                  image: DecorationImage(
                    image: AssetImage(
                      'assets/images/google_signin_button.png',
                    ),
                    fit: BoxFit.cover,
                  ),
                ),
              ),
            )
          ],
        ),
      ),
    );
  }

  @override
  Widget build(BuildContext context) {
    return isAuth ? buildAuthScreen() : buildUnAuthScreen();
  }
}
`

**Comment by Nic008888** [comment]
@harrykrish-web did you figure this out? I'm getting the same error trying to login with Facebook Auth package.

> Hi, i am facing same issue
> ![screen](https://user-images.githubusercontent.com/60507974/89683880-f750d880-d916-11ea-898d-c5ce113b7583.jpg)
> 
> Flutter Doctor Output:
> [✓] Flutter (Channel stable, v1.17.5, on Linux, locale en_GB.UTF-8)
> • Flutter version 1.17.5 at /home/harik/snap/flutter/common/flutter
> • Framework revision [8af6b2f](https://github.com/flutter/flutter/commit/8af6b2f038c1172e61d418869363a28dffec3cb4) (5 weeks ago), 2020-06-30 12:53:55 -0700
> • Engine revision ee76268252
> • Dart version 2.8.4
> 
> [✓] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
> • Android SDK at /home/harik/Android/Sdk
> • Platform android-30, build-tools 30.0.1
> • Java binary at: /home/harik/installations/android-studio/jre/bin/java
> • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
> • All Android licenses accepted.
> 
> [✓] Android Studio (version 4.0)
> • Android Studio at /home/harik/installations/android-studio
> • Flutter plugin version 47.1.2
> • Dart plugin version 193.7361
> • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
> 
> [✓] VS Code (version 1.47.3)
> • VS Code at /usr/share/code
> • Flutter extension version 3.13.2
> 
> [✓] Connected device (1 available)
> • SM G960F • 2311939b1b057ece • android-arm64 • Android 10 (API 29)
> 
> • No issues found!

**Comment by Nic008888** [comment]
This fixed the issue for me:

https://github.com/flutter/flutter/issues/15976

**Comment by CindyJ6** [comment]
i got the same issue please i need help

**Comment by pacholoamit** [comment]
Same here, getting redirected to message_codecs.

**Comment by d9media** [comment]
Getting the same error

**Comment by Muzamil132** [comment]
same here anyone know the solution please

**Comment by YashRenwa** [comment]
Guys! I got the same error and I got the solution by creating SHA1 Key in project settings in flutter. 
If you are on Mac, open terminal and type the command:
keytool -list -v \
-alias androiddebugkey -keystore ~/.android/debug.keystore
If you are on Windows:
keytool -list -v \
-alias androiddebugkey -keystore %USERPROFILE%\.android\debug.keystore

You will get SHA1 and SHA256 keys and then paste it in your app in firebase under project settings.
Hope it helps
